collectors/python.d.plugin/postgres/postgres.chart.py

```diff
@@ -650,6 +662,43 @@
 """,
 }
 
+QUERY_EMERGENCY_AUTOVACUUM = {
+    DEFAULT: """
+WITH max_age AS (
+    SELECT setting AS autovacuum_freeze_max_age
+        FROM pg_catalog.pg_settings
+        WHERE name = 'autovacuum_freeze_max_age' )
+, per_database_stats AS (
+    SELECT datname
+        , m.autovacuum_freeze_max_age::int
+        , age(d.datfrozenxid) AS oldest_current_xid
+    FROM pg_catalog.pg_database d
+    JOIN max_age m ON (true)
+    WHERE d.datallowconn )
+SELECT max(ROUND(100*(oldest_current_xid/autovacuum_freeze_max_age::float))) AS percent_towards_emergency_autovac
+FROM per_database_stats;
+""",
+}
+
+QUERY_TX_WRAPAROUND = {
+    DEFAULT: """
```

Comment:
Since you've added only `DEFAULT`, is this query valid for all versions?

Reply:
I verified these changes work against 9.6, 10, 11, 12, and 13.

Reply:
As I can see all columns exists on supported version. It is good for me.